frontend/src/views/Game.tsx

```diff
@@ -150,27 +151,38 @@ function GamePage() {
   };
 
   const createCodeLanguageArray = () => {
-    while (currentLanguage.length < problems.length) {
-      currentLanguage.push(Language.Python);
+    while (languageList.length < problems.length) {
+      languageList.push(Language.Python);
     }
 
-    while (currentCode.length < problems.length) {
-      currentCode.push('');
+    while (codeList.length < problems.length) {
+      codeList.push('');
```

Comment:
This later changes to default code, correct?

Reply:
Yes. This is just so that the code can load in the default code into the array without causing an IndexOutOfBounds exception.